layouts/partials/hooks/head-end.html

```diff
@@ -13,3 +13,13 @@
 {{- if .HasShortcode "cncf-landscape" -}}
 <script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.3.9/iframeResizer.min.js" integrity="sha384-hHTwgxzjpO1G1NI0wMHWQYUxnGtpWyDjVSZrFnDrlWa5OL+DFY57qnDWw/5WSJOl" crossorigin="anonymous"></script>
 {{- end -}}
+
+{{ if .HasShortcode "mermaid" }}
+  <!-- Copied from https://unpkg.com/mermaid@10.6.1/dist/mermaid.min.js -->
+  {{- with resources.Get "js/mermaid-10.6.1.min.js" -}}
+  <script src="{{ .RelPermalink }}" integrity="sha384-+NGfjU8KzpDLXRHduEqW+ZiJr2rIg+cidUVk7B51R5xK7cHwMKQfrdFwGdrq1Bcz"></script>
+  {{- else -}}
+    <!-- without Mermaid, the site won't appear right -->
```

Comment:
```suggestion
    {{/* Without Mermaid, the site won't appear right */ -}}
```

Reply:
I thought about that, but I went for an HTML comment. The build is about to fail in a few microseconds so I'm not worried about this comment appearing inside the `<head>` of the final rendered page.

Reply:
(see the very next line!)